docs/tutorial/part-one/index.md

```diff
@@ -48,10 +48,11 @@ To install a starter, first install Gatsby's command line program by running the
 npm install --global gatsby-cli
```

Comment:
If we want to list “open a new Terminal window”, maybe it should be listed here. Also, should we call it “terminal window”? I’m not sure whether “terminal” is a generic term for Unix terminal that applies to Windows and Linux as well, or if it comes from macOS’s “Terminal”

Reply:
good point. I honestly have no idea how the command line looks or works in Windows and Linux! Do you know?

Reply:
No idea, but I’ll google it so we don’t bother Kyle hehe.